ShadowsOfWar/UltraSpeakerMergePreReqs.cs

```diff
@@ -0,0 +1,51 @@
+/*
+name: Ultra Speaker Merge PreReqs
+description: Gets the prerequisites for the Ultra Speaker merge.
+tags: ultra speaker merge, ultra malgor merge
+*/
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/Story/ShadowsOfWar/CoreSoW.cs
+//cs_include Scripts/ShadowsOfWar/CoreSOfWar.cs
+using Skua.Core.Interfaces;
+
+public class UltraSpeakerMergePreReqs
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreSoW SoW = new();
+    public CoreSOfWar SOfWar = new();
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        GetPrereqs();
+
+        Core.SetOptions(false);
+    }
+
+    public void GetPrereqs()
+    {
+        SoW.CompleteCoreSoW();
+
+        SOfWar.DragonsTear();
+        SOfWar.Acquiescence(60);
+        SOfWar.ElementalCore(81);
+        //50 Brilliant Aura
```

Comment:
50 Brilliant Aura: CoreBLoD => FindingFragmentsMace

1 Blinding Aura: CoreBLoD => `FindingFragments(Core.CheckInventory("Blinding Scythe of Destiny") ? 2177 : 2174, "Blinding Aura");`

250 Enchanted Scale: `Other\Classes\DragonslayerGeneral.cs` =>            DSG.EnchantedScaleandClaw(quant, 0);

30 Dragon Scale (there are 2 types here so ItemID required*):
`while (!Bot.ShouldExit && !Core.CheckInventory(11475, 30))`
	`Core.KillMonster("lair", "Hole", "Center", "*", isTemp: false, log: false);`

75 Ultimate Darkness Gem: `Core.HuntMonster("shadowfallwar", "Skeletal Fire Mage", "Ultimate Darkness Gem", 50, isTemp: false);`

Death's Oversight: `Core.KillMonster("shadowattack", "Boss", "Left", "Death", "Death's Oversight", 5, false);`

25 Fire Avatar's Favor:
`Other\FireAvatarFavorFarm.cs` => FAFavor(25)

13 Fragment of the Queen:
`Core.HuntMonster("transformation", "Queen of Monsters", "Fragment of the Queen", 13, false, publicRoom: true);`


250 ShadowChaos Mote: 

`Core.EquipClass(ClassType.Farm);`
`Core.RegisterQuests(7700);`
`while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))`
`{`
`   Core.HuntMonster("lagunabeach", "Flying Fisheye|ShadowChaos Brigand", "Chaos-ShadowFlame`
`Sample", 15);`
`    Bot.Wait.ForPickup(req.Name);`
`}`
`Core.CancelRegisteredQuests();`

100 Warfury Emblem:
`Other\WarFuryEmblem.cs` => `WarfuryEmblemFarm(100);`

3 Purified Undead Dragon Essence:
`Good\BLoD\2UltimateBlindingLightofDestiny.cs` => `uBLOD.PurifiedUndeadDragonEssence();`

Ascended BoA:
`Chaos\AscendedDrakathGear.cs` =>  `AscendedGear("Ascended Blade of Awe");`

50 Ice Shard:
`Story\DragonFableOrigins.cs` (story for material quest)

`DFO.DragonFableOriginsAll();`
`Core.EquipClass(ClassType.Solo);`

`Core.AddDrop("Ice Shard");`
`while (!Bot.ShouldExit && !Core.CheckInventory(43712, 50))`
`{`
`    Core.EnsureAccept(6311);`
`    Core.HuntMonster("northmountain", "Izotz", "Ice Crystal");`
`    Core.EnsureComplete(6311);`
`    Bot.Wait.ForPickup("Ice Shard");`
`}`

Hollowborn Doomblade:
`Hollowborn\HollowbornDoomKnight\CoreHollowbornDoomKnight.cs` => `ADKFalls();`

hope this helps you a bit